alphatwirl/loop/DatasetIntoEventBuildersSplitter.py

```diff
@@ -78,7 +78,13 @@ def _file_nevents_list_for(self, dataset, maxEvents = -1, maxFiles = -1):
         for f in files:
             if 0 <= maxEvents <= totalEvents:
                 return ret
-            n = self.eventBuilderConfigMaker.nevents_in_file(f)
+            try:
+                n = self.eventBuilderConfigMaker.nevents_in_file(f)
+            except AttributeError:
+                import logging
```

Comment:
It might be safer to put this import higher up, outside of this block.  I guess logging is a standard library by now, but if it fails to import you'll get a pretty confusing traceback here.